packages/vira/src/Vira/Page/JobPage.hs

```diff
@@ -143,8 +145,32 @@ triggerNewBuild repoName branchName = do
   asks App.supervisor >>= \supervisor -> do
     job <- App.update $ St.AddNewJobA repoName branchName branch.headCommit supervisor.baseWorkDir
     log Info $ "Added job " <> show job
-    let stages = getStages repo branch mCachix mAttic
-    Supervisor.startTask supervisor job.jobId job.jobWorkingDir stages $ \result -> do
+    let
+      -- TODO: Get the settings from the downstream repo
+      getRepoSettings :: RepoSettings
+      getRepoSettings =
+        if repoName == "euler-lsp"
+          then
+            RepoSettings
+              ( [ Stage [] (Clone repo branch)
+                , Stage [BranchMatches "release-*"] (Build (BuildSettings ["--", "--override-input", "flake/local", "github:boolean-option/false"]))
```

Comment:
Should add a comment here for the `flake/` workaround, pointing to https://github.com/juspay/omnix/issues/452